### PR TITLE
[MI-2156] Add test cases for reducers

### PR DESCRIPTION
**Pull request by ayusht2810**

### CODE REVIEW

File: webapp/src/reducers/addSubscriptionModal/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import reducer, {showModal, hideModal, SubscriptionModalState} from 'reducers/addSubscriptionModal';
+
+test('should change state of open to "true"', () => {
```
Comment (Kshitij-Katiyar) [inline webapp/src/reducers/addSubscriptionModal/index.test.tsx:3]: do this everywhere
```suggestion
test('should change the state of open to "true"', () => {
```

File: webapp/src/reducers/connectedState/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import reducer, {setConnected, ConnectedState} from 'reducers/connectedState';
+
+test('should change state of connected to true on payload value equal to "true"', () => {
```
Comment (Kshitij-Katiyar) [inline webapp/src/reducers/connectedState/index.test.tsx:3]: is this make more sense?
```suggestion
test('should change the state of connected to true when payload value is "true"', () => {
```

**Review by Kshitij-Katiyar — APPROVED** [review]

File: webapp/src/reducers/addSubscriptionModal/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import reducer, {showModal, hideModal, SubscriptionModalState} from 'reducers/addSubscriptionModal';
+
+test('should change the state of open to "true"', () => {
+    const previousState: SubscriptionModalState = {
+        open: false,
+    };
```
Comment (manojmalik20) [inline webapp/src/reducers/addSubscriptionModal/index.test.tsx:6]: Define this outside the test.

File: webapp/src/reducers/addSubscriptionModal/index.test.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    expect(reducer(previousState, showModal())).toEqual(
+        {open: true},
+    );
+});
+
+test('should change the state of open to "false"', () => {
+    const previousState: SubscriptionModalState = {
+        open: true,
+    };
```
Comment (manojmalik20) [inline webapp/src/reducers/addSubscriptionModal/index.test.tsx:15]:
```suggestion
    previousState.open = false;
```

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**